config/environment.js

```diff
@@ -28,17 +28,37 @@ const envvars = {
 /**
  * @description
  * Browser list for autoprefixer, see https://github.com/ai/browserslist.
+ * Support output lists can be viewed via copy/pasting into http://browserl.ist.
```

Comment:
To make our lives a little easier, add the browserlist permalink to the comment: https://goo.gl/AXDSxG

Reply:
I didn't because that will totally be a URL that will be overlooked if the browserlist gets updated.

Reply:
It's true but it took me a few minutes to figure out how to input values into the site (comma separated list of values *without* the single quotes).

Reply:
I'll leave it for now, I'm not keen on obfuscating the domain either, and the full URL is really long. It'll be a minor inconvenience, but better than before where the domain wasn't in the comments at all.